x-pack/test/api_integration/apis/entity_manager/definitions.ts

```diff
@@ -27,43 +33,78 @@ export default function ({ getService }: FtrProviderContext) {
   describe('Entity definitions', () => {
     describe('definitions installations', () => {
       it('can install multiple definitions', async () => {
-        await installDefinition(supertest, mockDefinition);
-        await installDefinition(supertest, mockBackfillDefinition);
+        await installDefinition(supertest, { definition: mockDefinition });
+        await installDefinition(supertest, { definition: mockBackfillDefinition });
 
         const { definitions } = await getInstalledDefinitions(supertest);
         expect(definitions.length).to.eql(2);
         expect(
-          definitions.find(
+          definitions.some(
             (definition) =>
               definition.id === mockDefinition.id &&
               definition.state.installed === true &&
               definition.state.running === true
           )
-        );
+        ).to.eql(true);
         expect(
-          definitions.find(
+          definitions.some(
             (definition) =>
               definition.id === mockBackfillDefinition.id &&
               definition.state.installed === true &&
               definition.state.running === true
           )
-        );
+        ).to.eql(true);
 
-        await uninstallDefinition(supertest, mockDefinition.id);
-        await uninstallDefinition(supertest, mockBackfillDefinition.id);
+        await Promise.all([
+          uninstallDefinition(supertest, { id: mockDefinition.id, deleteData: true }),
+          uninstallDefinition(supertest, { id: mockBackfillDefinition.id, deleteData: true }),
+        ]);
       });
 
       it('does not start transforms when specified', async () => {
-        await installDefinition(supertest, mockDefinition, { installOnly: true });
+        await installDefinition(supertest, { definition: mockDefinition, installOnly: true });
 
         const { definitions } = await getInstalledDefinitions(supertest);
         expect(definitions.length).to.eql(1);
         expect(definitions[0].state.installed).to.eql(true);
         expect(definitions[0].state.running).to.eql(false);
 
-        await uninstallDefinition(supertest, mockDefinition.id);
+        await uninstallDefinition(supertest, { id: mockDefinition.id });
+      });
+    });
+
+    describe('definitions update', () => {
```

Comment:
Maybe good to add a test for trying to update a managed definition and when a change is already in progress but that may be hard to get stable due to timings

Reply:
can add a test for managed definition. the latter sounds like a recipe for flakiness